connection-service/azuredeploy.json

```diff
@@ -105,7 +105,7 @@
     "appGwSGPoolID": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/applicationGateways/', variables('appGwName'), '/backendAddressPools/', variables('appGwSGPoolName'))]",
     "appGwCSPoolName": "CSBackendPool",
     "appGwCSPoolID": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/applicationGateways/', variables('appGwName'), '/backendAddressPools/', variables('appGwCSPoolName'))]",
-    "cmnsgName": "[concat('cmNSG',parameters('CSUniqueSuffix'))]",
+    "sgnsgName": "[concat('sgNSG',parameters('CSUniqueSuffix'))]",
```

Comment:
sgNsgName? sgnsgName isn't super obvious to me what it is

Reply:
good point.